### PR TITLE
Add Richards seminar talk

### PR DESCRIPTION
**Pull request by rantahar**
Review:
 - Date and time
 - Re-read the abstract

### CODE REVIEW

**Review by lucaferranti — APPROVED** [review]

File: content/events/seminar-series/_index.md [hunk 1]
```diff
@@ -41,6 +41,18 @@ The Nordic-RSE team will provide support and infrastructure.
 We will publish upcoming seminar topics and abstracts here as soon as they are confirmed.
 You can see topics in planning and add your own suggestions on our [Issues page](https://github.com/nordic-rse/nordic-rse.github.io/issues).
 
+### March 2023: What a Research Software Engineering group at a Nordic university look like
+
+- Speaker: Richard Darst, [Aalto Scicomp](https://scicomp.aalto.fi/)
+- 2023-03-15, 13:00 CET [convert to your time zome](https://arewemeetingyet.com/Helsinki/2023-03-15/14:00)
+- Hackmd (contains connection details): <https://hackmd.io/@nordic-rse/seminar-march-2023>
+- Abstract:What does a Research Software Engineering (RSE) group at a Nordic university look like?  Aalto RSE supports the whole university, and Richard Darst and other Aalto RSE members will discuss the history behind their team, the way it works, future prospects, and lessons for others. This talk will focus on the administrative side of things, and discussion will focus on what one should know to reproduce this work at other universities.
```
Comment (mtomasini) [inline content/events/seminar-series/_index.md:49]: Space between "Abstract:" and "What"? Also, it seems to have two spaces between the end of the first sentence and "Aalto".

File: content/events/seminar-series/_index.md [hunk 2]
```diff
@@ -41,6 +41,18 @@ The Nordic-RSE team will provide support and infrastructure.
...
+Aalto University is the leading technical university in Finland. Started in 2020, Aalto RSE now serves the entire university in computing, data, and software problems.  They work as part of Science-IT, which is effectively the local "HPC team".  They have good connections to the local IT Services, Data Agents, and other research services.
```
Comment (mtomasini) [inline content/events/seminar-series/_index.md:50]: Are there always double spaces between end and starts of sentences? Seems a double space between "problems." and "They" and also between ""HPC team"." and "They".

**Review by mtomasini — COMMENTED** [review]
Couple of typos

**Comment by rkdarst** [comment]
I guess it's better to postpone due to the coderefinery install time...

**Comment by rantahar** [comment]
Updated time

**Comment by rantahar** [comment]
I suppose this is checked now and can be merged

File: content/events/seminar-series/_index.md [hunk 3]
```diff
@@ -41,6 +41,18 @@ The Nordic-RSE team will provide support and infrastructure.
 We will publish upcoming seminar topics and abstracts here as soon as they are confirmed.
 You can see topics in planning and add your own suggestions on our [Issues page](https://github.com/nordic-rse/nordic-rse.github.io/issues).
 
+### March 2023: What a Research Software Engineering group at a Nordic university look like
```
Comment (rkdarst) [inline content/events/seminar-series/_index.md:44]:
```suggestion
### April 2023: What a Research Software Engineering group at a Nordic university look like
```

**Pull request merged**